lib/Dialect/Poly/IR/PolyOps.cpp

```diff
@@ -17,14 +19,30 @@ LogicalResult FromTensorOp::verify() {
   auto tensorShape = getInput().getType().getShape();
   auto polyDegree = getOutput().getType().getRing().getIdeal().getDegree();
```

Comment:
nit. add `ring` variable, to avoid lengthy declaration of `polyDegree` & `coeffModulus`.

Reply:
Done!